sql/updates/world/4.3.4/2021_07_23_00_world.sql

```diff
@@ -0,0 +1,360 @@
+INSERT INTO `conditions` VALUES (15, 6944, 1, 0, 0, 47, 0, 26998, 8, 0, 0, 0, 0, '', 'Bat Handler Maggotbreath - Show gossip if quest 26998 is active');
+INSERT INTO `conditions` VALUES (18, 44894, 83756, 0, 0, 47, 0, 27045, 8, 0, 0, 0, 0, '', 'Allow spellclick if quest 27045 is active');
+INSERT INTO `conditions` VALUES (13, 1, 83464, 0, 2, 31, 0, 3, 44737, 0, 0, 0, 0, '', 'Chain Visual Left - Target Subdued Forest Ettin');
+INSERT INTO `conditions` VALUES (13, 1, 83467, 0, 2, 31, 0, 3, 44737, 0, 0, 0, 0, '', 'Chain Visual Right - Target Subdued Forest Ettin');
+INSERT INTO `conditions` VALUES (13, 1, 83763, 0, 0, 31, 0, 3, 44883, 0, 0, 0, 0, '', 'Armoire Camera - Target Crowley');
+INSERT INTO `conditions` VALUES (13, 1, 83764, 0, 0, 31, 0, 3, 44884, 0, 0, 0, 0, '', 'Armoire Camera - Target Ivar');
+
+ -- Dark Ranger
+SET @CGUID := 399000;
+INSERT INTO `creature` VALUES (@CGUID, 44632, 0, 130, 5369, 1, 0, 1, 169, 0, -1, 0, 0, 1360.47, 1063.27, 52.9999, 3.92421, 300, 0, 0, 1, 0, 2, 0, 0, 0, '', 0);
+INSERT INTO `creature` VALUES (@CGUID+1, 44632, 0, 0, 0, 1, 0, 1, 169, 0, -1, 0, 1, 1428.24, 1032.64, 53.0474, 4.06429, 300, 0, 0, 1398, 0, 2, 0, 0, 0, '', 0);
+
+INSERT INTO `creature_addon` VALUES (@CGUID, 446320, 0, 0, 0, 2, 0, 0, 0, 0, 0, '');
+INSERT INTO `creature_addon` VALUES (@CGUID+1, 446321, 0, 0, 0, 2, 0, 0, 0, 0, 0, '');
+
+ -- High Warlord Cromush
+INSERT INTO `creature` VALUES (@CGUID+2, 44640, 0, 0, 0, 1, 0, 1, 169, 0, -1, 0, 1, 1357.21, 1046.36, 52.7554, 4.46892, 300, 0, 0, 387450, 0, 2, 0, 0, 0, 'npc_high_warlord_cromush_44640', 0);
+
+INSERT INTO `creature_addon` VALUES (@CGUID+2, 446400, 0, 0, 0, 1, 0, 0, 0, 0, 3, '80697');
+
+ -- Worg
+SET @ENTRY := 1765;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
```

Comment:
did you ensure that these creatures are not spawned outside of silverpine?

Reply:
I did, actually! I can review it again in any case.

Reply:
They're not spawned anywhere else, so the row is ok!